gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/DefaultIdentityProviderServiceImpl.java

```diff
@@ -154,6 +156,9 @@ public Map<String, Object> createProviderConfiguration(String referenceId, NewId
             configMap.put("port", Integer.parseInt(jdbcPort()));
             configMap.put("database", jdbcDatabase());
             configMap.put("protocol", jdbcDriver());
+            if(DRIVERS_SUPPORTING_SCHEMA.contains(jdbcDriver()) && jdbcSchema() != null) {
+                configMap.put("schema", jdbcSchema());
+            }
```

Comment:
suggestion: add an option in the Options attribute of the configuration instead creating a dedicated field